app/styles/partials/overrides.scss

```diff
@@ -35,16 +35,38 @@ body.dimmable.undetached.dimmed {
   }
 }
 
-.margin.less {
-  padding: 0 !important;
+.less {
+  &.margin {
+    margin: 0 !important;
+  }
+
+  &.bottom.margin {
+    margin-bottom: 0 !important;
+  }
+
+  &.top.margin {
+    margin-top: 0 !important;
+  }
 }
 
 .less {
   &.padding {
     padding: 0 !important;
   }
-  
-  &.padding-right {
+
+  &.right.padding {
     padding-right: 0 !important;
   }
+
+  &.left.padding {
+    padding-left: 0 !important;
+  }
+}
+
+.ui.avatar.image {
```

Comment:
This is a standard semantic UI class, why have you made it generic for all ? Please wrap it
https://semantic-ui.com/elements/image.html